src/main.c

```diff
@@ -1778,7 +1778,14 @@ static void cmdline_handler(int argc, char **argv)
 		{CONVERT, "convert", "convert <INBUNDLE> <OUTBUNDLE>", "Convert to casync index bundle and store", convert_start, convert_group, FALSE},
 		{CHECKSUM, "checksum", "checksum <DIRECTORY>", "Deprecated", checksum_start, NULL, FALSE},
 		{INFO, "info", "info <FILE>", "Print bundle info", info_start, info_group, FALSE},
-		{STATUS, "status", "status", "Show system status", status_start, status_group, TRUE},
+		{STATUS, "status", "status",
+			"Show system status\n\n"
+			"List of status commands: (default image is the currently booted image)\n"
```

Comment:
I would prefer _subcommands_ instead of _commands_ (knowing that we have to fix that at different places in the documentation as well).
The `rauc status mark-*` subcommands act on _slots_, not on _images_.

Reply:
I'd agree with that and would add that the colon should be at the end of the line

Reply:
Correct terminology is important I agree, oversight on my part (slots vs images)
Addressed the others as well :)